static/image/cloud-lightning-white.svg

```diff
@@ -0,0 +1,14 @@
+<svg
```

Comment:
@siggisim LMK if this icon seems good for the sidebar -- a few options from feather icons I thought could work were:

* cloud-lightning (I thought it was kinda fitting because the builds happen in the cloud and they are "lightning fast", but this could be too abstract)
* tool (evokes too many feelings of manual configuration IMO)
* fast-forward (lets you "speed up" / "skip steps" in your dev process, but maybe too abstract)
* zap (kinda seems like it's destroying / deleting stuff or something)

Reply:
What do you think about `play-circle` https://feathericons.com/?query=play-circle?

Kinda similar to what Github uses for actions https://github.com/buildbuddy-io/buildbuddy/actions

Reply:
Good call, the metaphor (mostly) makes sense and it has recognition value -- done

Reply:
Ah good call, I sorta get the metaphor and it also has recognition value -- done